huggingface/tensorflow/buildspec.yml

```diff
@@ -30,14 +30,14 @@ images:
   BuildHuggingFaceTensorflowGpuPy37Cu110TrainingDockerImage:
     <<: *TRAINING_REPOSITORY
     build: &HUGGINGFACE_TENSORFLOW_GPU_TRAINING_PY3 false
-    image_size_baseline: &IMAGE_SIZE_BASELINE 8489
+    image_size_baseline: &IMAGE_SIZE_BASELINE 10500
```

Comment:
It seems that the base image caused this to get a lot bigger - how large are the packages installed on top of the DLC? @ahsan-z-khan - is this too big for SM?

Reply:
We additionally only installing `transformers` and 3 more python packages they should be < 200MB